src/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <AdditionalPackageTags>Hosting Aspire MassTransit RabbitMQ</AdditionalPackageTags>
+      <Description>A .NET Aspire integration for hosting MassTransit RabbitMQ services.</Description>
+       <ImplicitUsings>enable</ImplicitUsings>
+       <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting.RabbitMQ" Version="9.0.0"/>
```

Comment:
Please ensure packages are added using NuGet tooling so the package version is registered in the `Directory.Package.props`.

Also, for Aspire packages, use the variable available for the version.